docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -10,10 +10,120 @@ default:: `syn-vshn-sli-reporting`
 
 The namespace in which to deploy this component.
 
+== `images`
+
+[horizontal]
+type:: dict
+default:: See https://github.com/vshn/component-vshn-sli-reporting/blob/master/class/defaults.yml[`class/defaults.yml`]
+
+A list of images to be used by the component.
+
+== `auth.user`
+
+[horizontal]
+type:: string
+default:: "admin"
+
+Username for HTTP basic authentication required by the SLI reporting API.
+
+== `auth.password`
+
+[horizontal]
+type:: string
+
+Password for HTTP basic authentication required by the SLI reporting API.
+Required parameter.
+
+== `db_file`
+[horizontal]
+type:: string
+default:: "/data/data.db"
+
+Path of the sqlite database file to use.
+
+== `storage_requests`
+[horizontal]
+type:: string
+default:: "1Gi"
+
+Storage requests for the database volume.
+
+== `lieutenant.k8s_url`
+[horizontal]
+type:: string
+
+API URL where the kubernetes API of the Lieutenant cluster can be reached.
+Required.
+
+== `lieutenant.sa_token`
+[horizontal]
+type:: string
+
+Service account token which is able to get clusters on the Lieutenant cluster.
+Required.
+
+== `lieutenant.namespace`
+[horizontal]
+type:: string
+default:: lieutenant
+
+Namespace in which Cluster resources can be found on the Lieutenant cluster.
+
+== `network_policies.target_namespaces`
+
+[horizontal]
+type:: dictionary
+default:: `{}`
+example::
++
+[source,yaml]
+----
+appuio-reporting-database: true
+appuio-thanos: true
+----
+
+Injects a network policy into the target namespace given by the key, allowing communication to the namespace.
+
+[NOTE]
+This is a workaround and the parameter will disappear once we've a more native way to manage network policies with Project Syn.
+
+== `extra_volumes`
+
+[horizontal]
+type:: dict
+default:: {}
+
+A list of additional volumes to be mounted into the container.
```

Comment:
A quick description of `volume_spec` and `mount_spec` would be helpful here as it is not a standard kubernetes name/concept.